src/backend/columnar/Makefile

```diff
@@ -0,0 +1,22 @@
+citus_subdir = src/backend/columnar
+safestringlib_srcdir = $(citus_abs_top_srcdir)/vendor/safestringlib
+SUBDIRS = . safeclib
+SUBDIRS +=
+OBJS += \
+	$(patsubst $(citus_subdir)/%.c,%.o,$(foreach dir,$(SUBDIRS), $(sort $(wildcard $(citus_subdir)/$(dir)/*.c))))
+
+MODULE_big = columnar
+
+
+PG_CPPFLAGS += -I$(libpq_srcdir) -I$(safestringlib_srcdir)/include
+
+
+ifdef USE_PGXS
+PG_CONFIG = pg_config
+PGXS := $(shell $(PG_CONFIG) --pgxs)
+include $(PGXS)
+else
+subdir = src/backend/columnar
+top_builddir = ../../..
+include $(top_builddir)/Makefile.global
```

Comment:
Use `citus_top_builddir` in both of these lines.

Reply:
done